src/test/db_test.cc

```diff
@@ -263,3 +263,44 @@ TEST(DB, Iterator) {
   it->Seek("x");
   ASSERT_TRUE(!it->Valid());
 }
+
+TEST(DB, Get) {
+  zlog::Log *log;
+  auto be = new RAMBackend();
+  auto client = new FakeSeqrClient();
+  int ret = zlog::Log::Create(be, "log", client, &log);
+  ASSERT_EQ(ret, 0);
+
+  DB *db;
+  ret = DB::Open(log, true, &db);
+  ASSERT_EQ(ret, 0);
+
+  std::vector<std::string> strs{"a", "b", ""};
+  for (auto s : strs) {
+    auto txn = db->BeginTransaction();
+    txn->Put(s, s.empty() ? "Empty key" : s);
+    txn->Commit();
+  }
+
+  auto txn = db->BeginTransaction();
```

Comment:
Add a call to `db->Commit()` at the end for completeness. Since our transactions are somewhat limited at the moment, this is all good enough for a basic Get implementation. In the future we will need to record the set of keys read during a transaction to ensure they didn't change while the transaciton was executing, but that is future work.